backend/models/beneficiary_model.py

```diff
@@ -18,52 +17,59 @@
 class Beneficiary(Document):
     first_name = StringField(max_length=500)
     last_name = StringField(max_length=500)
-    phone = IntField(min_value=16, max_value=120)
-    landline = IntField(min_value=16, max_value=120)
+    phone = StringField()
+    landline = StringField()
     age = IntField(min_value=16, max_value=120)
-    zone = StringField(choices=[zone.value for zone in Zone], required=True)
+    zone = StringField(choices=[zone.name for zone in Zone], required=True)
     address = StringField(max_length=500, required=True)
-    special_condition = StringField(choices=[sc.value for sc in SpecialCondition], default="")
+    apartament = StringField(max_length=10)
+    scara = StringField(max_length=10)
```

Comment:
it is a good idea to use English for all fields name, maybe someone from Latvia will use our code :)

Reply:
I will rename, just moved the fields from the bottom.